src/test/java/seedu/address/model/patient/RecordTest.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.model.patient;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.RecordBuilder;
+
+public class RecordTest {
+
+    @Test
+    public void addDepartment() {
```

Comment:
Perhaps the method names for test methods should have been in the format `featureUnderTest_testScenario_expectedBehavior()`?  Otherwise the unit testing looks good and thorough :)

Reply:
Noted!